doc/manual/installation/installing-binary.xml

```diff
@@ -6,13 +6,119 @@
 
 <title>Installing a Binary Distribution</title>
 
-<para>If you are using Linux or macOS, the easiest way to install
-Nix is to run the following command:
+<para>If you are using Linux or macOS, the easiest way to install Nix
+is to run the following command:
 
 <screen>
-$ bash &lt;(curl https://nixos.org/nix/install)
+  $ bash &lt;(curl https://nixos.org/nix/install)
 </screen>
 
+If your system is supported by the multi-user installer, the
+installation tool will default to installing a multi-user Nix daemon.
+</para>
+
+<para>
+I you would like to explicitly opt in or out of the multi-user Nix
+installation, you can force the installer by passing additional
+arguments. See below.
+</para>
+
+<section xml:id="sect-multi-user-installation">
+  <title>Multi User Installation</title>
+  <para>
+    The multi-user Nix installation creates system users, and a system
+    service for the Nix daemon.
+  </para>
+
+  <itemizedlist>
+    <title>Supported Systems</title>
+
+    <listitem>
+      <para>Linux running systemd, with SELinux disabled</para>
+
+      <note><para>
+        If you have SELinux enabled, the installer will incorrectly
+        attempt to perform a multi-user installation. Please force a
+        <link linkend="sect-single-user-installation">single-user
+        installation</link>.
+      </para></note>
+    </listitem>
+    <listitem><para>macOS</para></listitem>
+  </itemizedlist>
+
+  <para>
+    You can force the installer to attempt a multi-user installation on
+    your system, even if it is unsupported:
+
+    <screen>
+  curl https://nixos.org/nix/install | sh -s -- --daemon
```

Comment:
Won't this give

```
No TTY, assuming you would say yes :)
```

regarding the y/n questions in the multi-user installer? That seems like a bad thing to recommend by default.

Reply:
You're right!

Reply:
Fixed by your recommendation on IRC, thank you!